src/osp/tasks/execute.cpp

```diff
@@ -525,6 +526,7 @@ static void pipeline_advance_reqs(Tasks const& tasks, TaskGraph const& graph, Ex
                 return true; // On the right stage and task not running. This means it's done
             }
         }();
+        // NOLINTEND(ugprone-branch-clone)
```

Comment:
`ugprone-branch-clone` -> `bugprone-branch-clone`

This check seems kind of annoying? maybe disable it?

Reply:
imho it's a useful check, just not in this specific situation.